server/controllers/NeedController.js

```diff
@@ -0,0 +1,55 @@
+const {createNeed, getNeed, deleteNeed} = require('../store/needs');
+const {deleteBidsForNeed} = require('../store/bids');
+const {createMission} = require('../store/missions');
+const {updateVehicleStatus} = require('../store/vehicles');
+const createConstraints = require('./constraints/need/create');
+
+// using downloaded validate.js file because latest version does not have type checking and docker fails with git source in package.json
```

Comment:
You can delete this comment.
It's interesting during the review time, and less so for posterity.
If you'd like future generations to understand your reasoning, maybe include it on the top of the validation file itself.

Reply:
Will include it in the validation file.